confection/util.py

```diff
@@ -134,3 +143,6 @@ def reverse(self, *args, **kwargs):
 
     def sort(self, *args, **kwargs):
         raise NotImplementedError(self.error)
+
+    def __deepcopy__(self, memo):
+        return self.__class__(deepcopy(v) for v in self)
```

Comment:
Not sure how to work memo into this - I don't see a way to set values after __new() 🤔